dp-hello-world-controller/routes/routes.go

```diff
@@ -2,18 +2,23 @@ package routes
 
 import (
 	"context"
+	"net/http"
 
 	"github.com/ONSdigital/dp-hello-world-controller/config"
 	"github.com/ONSdigital/dp-hello-world-controller/handlers"
 
-	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
-	"github.com/ONSdigital/log.go/log"
+	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/gorilla/mux"
 )
 
+// Clients - struct containing all the clients for the controller
+type Clients struct {
+	HealthCheckHandler func(w http.ResponseWriter, req *http.Request)
+}
+
 // Setup registers routes for the service
-func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, hc health.HealthCheck) {
-	log.Event(ctx, "adding routes")
-	r.StrictSlash(true).Path("/health").HandlerFunc(hc.Handler)
+func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, c Clients) {
+	log.Info(ctx, "adding routes")
+	r.StrictSlash(true).Path("/health").HandlerFunc(c.HealthCheckHandler)
 	r.StrictSlash(true).Path("/helloworld").Methods("GET").HandlerFunc(handlers.HelloWorld(*cfg))
```

Comment:
Again worth adding necessary TODO comments for things that will need to be removed such as this helloworld endpoint.

```suggestion
        // TODO remove helloworld endpoint once application has been initialised
	r.StrictSlash(true).Path("/helloworld").Methods("GET").HandlerFunc(handlers.HelloWorld(*cfg))
```

fyi - these are in here as an example of how to layout the app